public/sass/components/_panel_table.scss

```diff
@@ -19,6 +19,8 @@
   text-align: center;
   font-size: 90%;
   line-height: 2px;
+  background-color: ghostwhite;
```

Comment:
this wont work for light theme

Reply:
Hmmm,
it's like gray and I thought gray is good for both light and dark mode.

Reply:
If I understood @jessover9000 description in the original issue the background color should be the `$tight-form-func-bg;`